pallets/runtime/tests/src/bridge.rs

```diff
@@ -788,3 +788,40 @@ fn do_force_mint() {
         BridgeTxStatus::Handled
     );
 }
+
+#[test]
+fn genesis_txs() {
+    let alice = AccountKeyring::Alice.public();
+    let bob = AccountKeyring::Bob.public();
+    let one_amount = 111;
+    let two_amount = 222;
+    ExtBuilder::default()
+        .regular_users(vec![alice, bob])
+        .bridge_txs(vec![
+            BridgeTx {
+                nonce: 1,
+                recipient: alice,
+                amount: one_amount,
+                tx_hash: Default::default(),
+            },
+            BridgeTx {
+                nonce: 2,
+                recipient: bob,
+                amount: two_amount,
+                tx_hash: Default::default(),
+            },
+        ])
```

Comment:
Could you harden this test by expecting the full `(k1, k2, value)` list in `BridgeTxDetails` (after sorting for stability)?